tutugodfrey/ilearn
# #161571618 As a user I should be able to sign in

#### What does this pull request do?
- Implement user sign in with in-memory dummy data

#### Description of tasks to be completed
- Enable user sign in with username and password
- Ensure that fields are validated before sign in
 #### How should this be manually tested
- Make a local copy of the branch install dependencies and run `npm run start:dev` to start the dev server
- Using postman test user sign in with the schema below
```
{
"username": "",
"password": ""
}
```
- Confirm can you cannot sign in a user that does not exist and the fields cannot be empty when for sign in.

#### What are the relevant Pivotal tracker stories
#161571618
